<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginDlg</class>
 <widget class="QDialog" name="CLoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #4F6BBF;
border-radius:5px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="le_userName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;
font: 13pt &quot;宋体&quot;;
background:#267BFC;
border-radius:17px;</string>
    </property>
    <property name="placeholderText">
     <string>请输入您的登录用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>海特币交易所管理后台</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;
font: 13pt &quot;宋体&quot;;
background:#267BFC;
border-radius:17px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入您的登录密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_googleCode">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;
font: 13pt &quot;宋体&quot;;
background:#267BFC;
border-radius:17px;</string>
    </property>
    <property name="placeholderText">
     <string>请输入谷歌验证码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Login">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">	color:#ffffff;
	background:#267BFC;
	font: 14pt &quot;新宋体&quot;;
	border:none;
	border-radius:17px;
	outline:none;

</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_Close">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
outline:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource/res.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
